### ISSUE TITLE
Node v14 not supported

### ISSUE THREAD

**Issue by vitaly-t** [voxpelli/node-connect-pg-simple#172]
To make it work right under Node v14, you need to update `pg` dependency to version 8.0.3, which is where compatibility with Node v14 was fixed.

**Comment by voxpelli** [comment]
Thanks!

Until then, and preferably also after, one can provide a connection pool oneself: https://github.com/voxpelli/node-connect-pg-simple#advanced-options If there's nothing breaking in `pg` 8.x then it should just work

**Comment by nstuyvesant** [comment]
I tried that but experienced problems...
```javascript
import session from 'express-session';
import { native as pgNative } from 'pg';
import ConnectPgSimple from 'connect-pg-simple'; // depends on pg < 8.0.3
import config from '../../config/environment';

const pgNativePool = new pgNative.Pool({
  max: 10, // default
  connectionString: config.pg.uri
});

// Persist Express sessions in PostgreSQL
const pgSession = new ConnectPgSimple(session);
const store = new pgSession({
  pgNativePool,
  tableName: 'sessions'
});

export default {
  query: (sql, params) => pgNativePool.query(sql, params),
  store // Hangs in Node 14
};
```

**Comment by nstuyvesant** [comment]
Added this pull request: https://github.com/voxpelli/node-connect-pg-simple/pull/174. All tests ran successfully.

**Comment by vitaly-t** [comment]
Seems ok.

**Comment by voxpelli** [comment]
I'll keep this open until I've tested and merged the PR 🙂

**Comment by zont** [comment]
We are really looking forward to this fix.

**Comment by voxpelli** [comment]
@zont I'll take a look at it now

**Comment by alxndrsn** [comment]
@nstuyvesant looks like your PR works - many thanks!

For any `yarn` users looking for a temporary fix:
```
yarn remove connect-pg-simple
yarn add https://github.com/nstuyvesant/node-connect-pg-simple.git#adcb6c4642db88e6de47e3aea48a71b5ab3569fa
```

**Comment by voxpelli** [comment]
Place where the Node 14 fix was made: https://github.com/brianc/node-postgres/pull/2171

**Comment by voxpelli** [comment]
Shipped in [`6.2.0`](https://github.com/voxpelli/node-connect-pg-simple/releases/tag/v6.2.0)
